extra/fuzzer/set-gc.cpp

```diff
@@ -0,0 +1,90 @@
+//
```

Comment:
I'm wondering if, now that we are gonna treat Fuzzers more seriously and maintain them actively, we shall for example do the same kind of tricks we do in the unit tests to reuse the same testing code for all the variations of the `MemoryPolicy` (gc, non gc, etc.)